python/ray/autoscaler/azure/node_provider.py

```diff
@@ -52,7 +52,7 @@ def __init__(self, provider_config, cluster_name):
                 client_class=NetworkManagementClient, **kwargs)
         except Exception:
             logger.info(
-                "CLI profile authentication failed. Trying MSI", exc_info=True)
+                "CLI profile authentication failed. Trying MSI")
```

Comment:
since we've catching a bare exception we should keep the exception trace in case something weird is happening

Reply:
reason I removed it is that it looks weird in the logs. maybe we should catch the proper exception?

Reply:
I'll try once my build finishes...

Reply:
catching proper exception